src/modules/file-preview-card.marko

```diff
@@ -0,0 +1,535 @@
+                                height="16"
+                                width="16"
+                                aria-hidden="true"
+                            >
+                                <use href="#icon-overflow-vertical-16"/>
+                            </svg>
+                        </button>
+                    </div>
+                    <div class="file-preview-card__preview-action">
+                        <button class="icon-btn" type="button">
+                            <svg
+                                class="icon icon--16 file-preview-card__video-icon"
+                                aria-hidden="true"
+                            >
+                                <use href="#icon-play-16"/>
+                            </svg>
+                            <span class="file-preview-card__preview-action-text">
+                                0:21
+                            </span>
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <highlight-code type="html">
+        <div class="file-preview-card">
+            <div class="file-preview-card__body">
+                <img
+                    src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-real-square-pic.jpg"
+                    alt="Image"
+                >
+            </div>
+            <div class="file-preview-card__menu-action">
+                <button class="icon-btn" type="button" aria-label="Open menu">
+                    <svg
+                        class="icon icon--16"
+                        height="16"
+                        width="16"
+                        aria-hidden="true"
+                    >
+                        <use href="#icon-overflow-vertical-16"/>
+                    </svg>
+                </button>
+            </div>
+            <div class="file-preview-card__preview-action">
+                <button class="icon-btn" type="button">
+                    <svg
+                        class="icon icon--16 file-preview-card__video-icon"
+                        aria-hidden="true"
+                    >
+                        <use href="#icon-play-16"/>
+                    </svg>
+                    <span class="file-preview-card__preview-action-text">
+                        0:21
+                    </span>
+                </button>
+            </div>
+        </div>
+    </highlight-code>
+
+    <h3>Non image</h3>
+
+    <p>
+        For non-image file types, such as documents, the card does not display a preview within its body.
+    </p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div style="height: 168px; width: 168px;">
+                <div class="file-preview-card">
+                    <div class="file-preview-card__body">
+                        <svg
+                            class="icon file-preview-card__general-icon"
```

Comment:
Should it be `file-preview-card__general-icon` or `file-preview-card__document-icon`? If general, the heading should be general also. Let's try and be consistent with the nomenclature.

Reply:
I forgot to switch these when I changed it, will fix. Thanks!